vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -295,14 +299,17 @@ class Validator(
      *
      * @param it The [IssuerSigned] structure from ISO 18013-5
      */
-    fun verifyIsoCred(it: IssuerSigned, issuerKey: CoseKey): Verifier.VerifyCredentialResult {
+    fun verifyIsoCred(it: IssuerSigned, issuerKey: CoseKey?): Verifier.VerifyCredentialResult {
```

Comment:
function called `verify` turns into NOOP with null key?

Reply:
it returns `InvalidStructure`, three lines below